src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/annotations/ARcaConf.java

```diff
@@ -11,7 +11,7 @@
  * therefore, it uses the rca.conf* files in the test/resources as defaults.
  */
 @Retention(RetentionPolicy.RUNTIME)
-@Target(ElementType.TYPE)
+@Target({ElementType.TYPE, ElementType.METHOD})
```

Comment:
I can't find this ElementType.METHOD anywhere else. what is the purpose of adding this annotation ?

Reply:
Because I want to use this `@ARcaConf` annotation to override config path in method level when testing `YoungGenHighThresholdTest.testMasterNodeThresholdNotBreached` method

Reply:
I see. make sense